src/toolkits/object_detection/object_detector.cpp

```diff
@@ -281,6 +281,14 @@ void object_detector::init_options(
       "If True, print progress updates and model details.",
       true,
       true);
+  options.create_categorical_option(
+      /* name              */ "model",
+      /* description       */
+      "Defines the model type",
+      /* default_value     */ "darknet-yolo",
+      /* allowed_values    */
+      {flexible_type("darknet-yolo"), flexible_type("other")},
```

Comment:
If we don't know yet what the other categorical values will be, for now we can just make this an arbitrary string value

Reply:
Done